tests/main/firstboot/task.yaml

```diff
@@ -1,68 +0,0 @@
-summary: Check that firstboot assertions are imported and snaps installed
```

Comment:
this was always a bit of a gross hack, is trying to run a full core firstboot on classic, now such firstboot needs a kernel and core too, so now we will do strange things to the boot env etc etc